File: exercícios/lista9/ex1.scala
object Esfera {

  def calcularVolume(raio: Double): Double = {
    val pi = 3.0
    (4.0 / 3.0) * pi * math.pow(raio, 3)
  }

  def calcularAreaSuperficie(raio: Double): Double = {
    val pi = 3.0
    4 * pi * math.pow(raio, 2)
  }

  def main(args: Array[String]): Unit = {
    val raio = 6.0

    val volume = calcularVolume(raio)
    val area = calcularAreaSuperficie(raio)

    println(f"Raio: $raio%.1f cm")
    println(f"Volume da esfera: $volume%.2f cm³")
    println(f"Área da superfície da esfera: $area%.2f cm²")
  }
}

File: exercícios/lista11/ex1.hs
volumeEsfera :: Float -> Float
volumeEsfera r = (4 / 3) * piAprox * r^3
  where piAprox = 3.0

areaSuperficieEsfera :: Float -> Float
areaSuperficieEsfera r = 4 * piAprox * r^2
  where piAprox = 3.0

main :: IO ()
main = do
  let raio = 6.0
  let volume = volumeEsfera raio
  let area = areaSuperficieEsfera raio

  putStrLn ("Raio: " ++ show raio ++ " cm")
  putStrLn ("Volume da esfera: " ++ show volume ++ " cm³")
  putStrLn ("Área da superfície da esfera: " ++ show area ++ " cm²")
